src/components/projects/CreateProjectForm.vue

```diff
@@ -397,6 +399,7 @@
             >
               <b-input
                 v-model="exposures[n-1].angle"
+                :class="addSymbol(exposures[n-1].angle)"
```

Comment:
that's so weird that setting `:class="angle-input"` makes the input disappear?!

Reply:
it's not the class, it's the content of the class. I'm trying to understand why that's the case but when the class is a method and the content is a symbol, it works 🤔

Reply:
hmmmmm